packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.width.tests.js

```diff
@@ -172,11 +172,11 @@ QUnit.module('Width', () => {
         });
     });
 
-    QUnit.test('Does not render navbuttons: dx-tabs { max-width: 413px; } .dx-tab{ width: 100px; min-width: 0; }', function(assert) {
+    QUnit.test('Does not render navbuttons: dx-tabs { max-width: 413px; } .dx-tab{ width: 100px; min-width: 0 !important; }', function(assert) {
```

Comment:
Hm, why did "!important" appear here?

Reply:
This test checks a custom script. The user needs to override the base min-width value. To avoid incrementing the selector, we use !important.